tests/posix/headers/src/stropts_h.c

```diff
@@ -26,3 +26,10 @@ ZTEST(posix_headers, test_stropts_h)
 	zassert_not_equal(RS_HIPRI, ~RS_HIPRI);
 	#endif
 }
+
+ZTEST(posix_headers, test_stropts_putpmsg)
+{
+	#ifdef CONFIG_POSIX_API
```

Comment:
we usually do not indent preprocessor macros

Reply:
Should there be any form of indentation? Or would it be
```
ZTEST(posix_headers, test_stropts_putpmsg)
{
#ifdef CONFIG_POSIX_API
```

Reply:
Yeah should be something like that